covid_graphs/covid_graphs/country_graph.py

```diff
@@ -45,9 +47,13 @@ def __init__(
         min_start_date_idx = report.dates.index(min_start_date)
         self.cropped_dates = report.dates[min_start_date_idx:]
         self.cropped_cumulative_active = report.cumulative_active[min_start_date_idx:]
+        self.display_at_least_until = max(
+            max(curve.display_at_least_until for curve in self.curves) + datetime.timedelta(days=1),
+            self.cropped_dates[-1] + EXTENSION_PERIOD,
+        )
 
     def create_country_figure(self, graph_type=GraphType.Linear):
-        log_xaxis_date_since = datetime.date(2020, 2, 1)
+        log_xaxis_date_since = self.cropped_dates[0] - datetime.timedelta(days=1)
```

Comment:
Did you test this? I think for some countries with small values (like Jordan I think) offsetting the x values by more looked better.

Reply:
Sidenote: In the future I can imagine more countries in one graph; having consistent axis will be then necessary. (But not needed for now.)